test/script/script-unit.js

```diff
@@ -78,7 +78,8 @@ describe('Script', function() {
       input.execute(stack);
       output.execute(stack);
 
-      assert.deepEqual(stack.items, [[1], [3], [5]]);
+      const expected = [Buffer.from([1]), Buffer.from([3]), Buffer.from([5])];
```

Comment:
These changes allow `npm test-browser` to pass without affecting `npm test`. Similar changes were bcoin here:
https://github.com/bcoin-org/bcoin/blob/01a71561b1677981a4b355f7deec46f50e68a855/test/script-test.js#L107